src/main/java/com/ticketservice/service/TicketService.java

```diff
@@ -37,5 +45,23 @@ public static void main(String[] args) {
         fullTicketUsingSetters.setStadiumSector('A');
         fullTicketUsingSetters.setMaxBackpackWeight(9.99);
         fullTicketUsingSetters.setPrice(new BigDecimal(5.99));
+
+        ticketService.save(emptyTicketUsingConstructor);
+        ticketService.save(limitedTicketUsingSetters);
+        ticketService.save(fullTicketUsingConstructor);
+
+        System.out.println("Search result: " + ticketService.findByStadiumSector('B'));
```

Comment:
Just a small reminder. Using System.out.println (often abbreviated as Sysout) for logging in production code is generally not recommended.